src/napari_spatialdata/_model.py

```diff
@@ -66,7 +66,7 @@ def get_items(self, attr: str) -> Tuple[str, ...]:
             if self.points_var is not None:
                 return tuple(map(str, self.points_var.unique()))
             else:
-                return tuple(["No points found."])  # noqa: C409
+                return tuple(["No points found."])
         return tuple(map(str, getattr(self.adata, attr).index))
```

Comment:
I feel `# noqa: C409` needs to be re added to the return otherwise it'd fail again.